src/Sylius/Behat/Page/Admin/Product/CreateSimpleProductPage.php

```diff
@@ -217,6 +217,19 @@ public function selectShippingCategory($shippingCategoryName)
         $this->getElement('shipping_category')->selectOption($shippingCategoryName);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function setShippingRequired($isShippingRequired)
+    {
+        if ($isShippingRequired) {
+            $this->getElement('shipping_required')->check();
+
+            return;
+        }
+
+        $this->getElement('shipping_required')->uncheck();
```

Comment:
What would you say for two methods, `requireShipping` and `doNotRequireShipping` (or something similar)? For now it could be even  only the second one, cause as far as I see there is only `->setShippingRequired(false)` used in code.